### ISSUE TITLE
Hanging at "Setting up ca-certificates-java" while install eclipse in Docker 1.13.1

### ISSUE THREAD

**Issue by premier424** [docker/docker#31026]
I'm trying to install the eclipse in docker container on Ubuntu 14.04 using apt-get and meet infinite pause after occurrence of  "Setting up ca-certificates-java (20130815ubuntu1)..." 

Similar issue is reported #18180 but it seems my issue is unable to be solved by update the kernel to 3.13.0-79.123 or later version since it is already the latest kernel version (4.2.0-27). 

**`docker version`:**

```
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:42:29 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 06:42:29 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**`docker info`:**

```
Server Version: 1.13.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 27
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1
runc version: 9df8b306d01f59d3a8029be411de015b7304dd8f
init version: 949e6fa
Security Options:
 apparmor
Kernel Version: 4.2.0-27-generic
Operating System: Ubuntu 14.04.4 LTS
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 3.86 GiB
Name: xinzhou-VirtualBox
ID: OHWO:3WVG:OEO3:QJMR:P436:FKA2:CP26:6PPF:XS4I:44LY:PFN4:KUVY
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional information you deem important (e.g. issue happens only occasionally):**
Please provide me some further information how to solve this issue. Thank you.

**Comment by AkihiroSuda** [comment]
Ubuntu seems no longer supporting kernel 4.2 for 14.04
https://wiki.ubuntu.com/Kernel/Support#A14.04.x_Ubuntu_Kernel_Support

Do you hit the issue with kernel 4.4?

**Comment by premier424** [comment]
Thanks for your help. I upgrade to kernel 4.4.49 and even unable to start the docker. I tried reinstall docker and use `sudo` as well. It shows the following error when I run `sudo docker images`

```Cannot connect to Docker daemon at unix://var/run/docker.sock. Is the docker daemon running.```

Otherwise, after I upgrade the kernel, my virtual box keep sending message of: ```VBoxClient: the VirtualBox kernel service is not running.  Exiting``` I'm not sure if it is related to the problem above but I just mention it in case. 

I'm still troubleshooting this problem and maybe will update later. Please let me know if you have any suggestion on it.

**Comment by AkihiroSuda** [comment]
Do you hit the issue with a newly installed environment? (I tested and could not hit)

For the issue about "Cannot connect to Docker daemon", `journalctl -u docker` would be helpful.
VBox does not seem related.

**Comment by premier424** [comment]
The issue is solved when I reinstall docker with version 1.6.2 and kernel version keeping at 4.4.49. Thank you so much.
